polaris_wyre/wyre/api.py

```diff
@@ -0,0 +1,68 @@
+from urllib.parse import urljoin
+
+import requests
```

Comment:
I see that this gets installed by another dependency. Still, since it's being used by the project, shouldn't it also be added to the requirements file?